Documentation/media/uapi/v4l/meta-formats.rst

```diff
@@ -19,6 +19,7 @@ These formats are used for the :ref:`metadata` interface only.
 .. toctree::
     :maxdepth: 1
 
+    pixfmt-meta-bcm2835-isp-stats
```

Comment:
The reason I did not add the documentation to that earlier patch was because the documentation pulls structures from the isp uapi header file - which gets added in the large patch.  Perhaps I should add the uapi header file as a separate first patch?